wiki/Tournaments/OUKT/5/en.md

```diff
@@ -629,7 +629,7 @@ The RO16 onwards will be a standard Double Elimination bracket, seeded by perfor
 ### Scheduling Rules
 
 - The tournament will aim to be on the weekends, as the majority of players will be available. However, matches can be scheduled on any day dependent on availability of staff/players.
-- If you cannot play at the time posted for you, contact the organizer for a re arrangement.
+- If you cannot play at the time posted for you, contact the organizer for a rearrangement.
 - No-Showing is subject to the rules outlined in Tournament Rules.
```

Comment:
isn't it "not showing"?

Reply:
This would bring it in line with the rest of the OUKT articles
```suggestion
- Not-Showing is subject to the rules outlined in Tournament Rules.
```

Doing a quick search of the wiki looks like "no showing" and "not showing" are equally used, "no show" is the most popular but wouldn't make sense here.

Reply:
alright